src/main/scala/cromwell/binding/types/WdlStringType.scala

```diff
@@ -12,6 +12,7 @@ case object WdlStringType extends WdlPrimitiveType {
     case s: String => WdlString(s)
     case s: JsString => WdlString(s.value)
     case s: WdlString => s
+    case f: WdlFile => WdlString(f.value)
```

Comment:
From a WDL-writer's point of view, is there a list of "must-be-supported" coercions in the WDL spec?